src/containers/shared/components/Transaction/NFTokenMint/Simple.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import { useTranslation } from 'react-i18next'
+import { SimpleRow } from '../SimpleRow'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { NFTokenMintInstructions } from './types'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps<NFTokenMintInstructions>) => {
+  const { tokenID, tokenTaxon, uri } = data.instructions
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <SimpleRow label={t('token_id')} className="dt" data-test="token-id">
+        {tokenID}
+      </SimpleRow>
+      <SimpleRow
+        label={t('token_taxon')}
+        className="dt"
```

Comment:
I remember another comment about `dt`, but I can't remember what it was about - does this already get covered by Simple Row?

Reply:
I created #361